Upgrade.md

```diff
@@ -0,0 +1 @@
+* Reload plugin manager listing on update error
```

Comment:
I would suggest to describe the problem a little more explicit like "Adds missing handling of plugin update fails by reloading the plugin listing in the plugin manager window".